src/test/storage/simd_bp128_test.cpp

```diff
@@ -82,6 +82,43 @@ TEST_P(SimdBp128Test, DecompressSequenceUsingIterators) {
   }
 }
 
+// Test that the iterator's advance() method works as expected. Creating a sufficiently large vector to ensure that
```

Comment:
Could you clarify what you are testing here that goes beyond SegmentIteratorsTest?

Reply:
Well, the segment iterator tests cover similar functionality, but for segment iterators. I guess you mean the vector compression tests? That's a good point. Will take a look to move the test.

Reply:
We would not guarantee to test all possible bit widths with other tests (Param of the SIMD tests). Thus I think the test is correct here.

Reply:
> I guess you mean the vector compression tests?

Yes

> We would not guarantee to test all possible bit widths with other tests (Param of the SIMD tests). Thus I think the test is correct here.

Works for me, I'd just clarify that.

Reply:
```
// This test tests (in addition to the advance tests in the CompressedVectorTest) large advances over meta blocks for
// varying bit sizes (GetParam()).
```

Ok?

Reply:
Maybe this:

Tests that advancing the iterators with step sizes > 1 works for SIMD-BP128 vectors. Supplements CompressedVectorTest in that different bit sizes are tested.